downstream/modules/automation-hub/proc-configure-automation-hub-server-cli.adoc

```diff
@@ -0,0 +1,65 @@
+[id="proc-configure-automation-hub-server-cli"]
+= Configuring Red Hat {HubName} as the primary source for content
+
+Configure Red Hat automation hub as your primary source of content to access Ansible Certified Content Collections. You can configure automation hub in the command-line interface (CLI) or the web console.
+
+== Configuring Red Hat automation hub as the primary source for content using the CLI
+
+Configure Red Hat automation hub as your primary source of content by using the command-line interface (CLI). To configure automation hub, you must modify the `ansible.cfg` configuration file. With automation hub, you have access to certified, supported collections.
+
+.Prerequisites
+
+* You have obtained the API token for the {HubName} server. See link: https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/1.2/html/getting_started_with_red_hat_ansible_automation_hub/proc-create-api-token[Creating the {HubName} API token] for more information.
+[IMPORTANT]
+====
+	Creating a new token revokes any previous tokens 
+	generated for Private Automation Hub. Ensure that you update any Controller or scripts that you created with the previous token.
+====
+
+.Procedure
+
+. Add the `server_list` option under the `[galaxy]` section and provide one or more server names.
+. Create a new section for each server name:
++
+-----
+[galaxy_server.<server_name>]
+-----
+
+. Set the `url` option for each server name. You must include the `api/galaxy/` subdirectory in the server URL:
++
+[subs="+quotes"]
+-----
+https://__<server_fully_qualified_domain_name>__/api/galaxy/
+-----
+. Set the `auth_url` option if necessary. The community Ansible Galaxy does not require an `auth_url`.
```

Comment:
Step 4 appears to be an optional step. I believe we have a specific formatting convention for optional steps (they should look something like "4. Optional: set the auth_url option..." See: https://docs.engineering.redhat.com/display/CCSENABLE/Optional+and+conditional+steps